### ISSUE TITLE
Access/edit animation without affecting current nodes

### ISSUE THREAD

**Issue by WeirdosKlamber** [godotengine/godot-proposals#3829]
### Describe the project you are working on

Action/sport game with complex main character

### Describe the problem or limitation you are having in your project

Whenever I access an animationplayer to edit an animation it adjusts existing nodes without even playing the animation
This is frustrating and time consuming when those nodes have been carefully adjusted in position,rotation and are now splayed all over the place with no Undo button!
Especially if you don't  notice and there is a subsequent autosave

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Ideally running animations in the editor would not affect the nodes' position etc. as I would think everyone would only need the animations to affect nodes during gameplay
This might require a separate viewing window or 'animation state' variable on the nodes, might be a lot of work

As a minimum it would be helpful if selecting an Animationplayer and the animations within did not affect any nodes until you press 'Play'
This would allow editing animations without affecting the nodes 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

When you select an Animationplayer or an animation within the animationplayer it instantly imposes the animation onto the nodes, the code that does this could be paused until 'Play' 

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No, not really as this is an issue while using the editor and GDScript is not running

### Is there a reason why this should be core and not an add-on in the asset library?

Counter-intuitive and frustrating

**Comment by Calinou** [comment]
This was already addressed by the [implementation of an automatically created RESET track](https://github.com/godotengine/godot-proposals/issues/1597) for newly created tracks. (It does not work retroactively with tracks created before Godot 3.4.)

**Comment by WeirdosKlamber** [comment]
That is a step forward (unfortunately not for my 3.3 project!) 
However I would think this still causes problems if you move nodes around after creating the animations and reset states, every move would have to be keyed back into the animations. This is essentially the limitation of my 'rest state' animation workaround, which I use like RESET but is perpetually out-of-date

It still seems a conceptual disjoint that animations playing (or even not playing, just selected) within the editor, change the nodes in the editor rather than just during runtime

**Comment by AnidemDex** [comment]
There's a proposal here that aims to integrate a new animation editor. That plugin animate nodes without modifying the original scene. Maybe that can help here too? https://github.com/godotengine/godot-proposals/issues/3950

**Comment by and-rad** [comment]
I agree with this proposal. I'm only just getting started with complex animations and cinematics in Godot, but I have a strong suspicion that the RESET track was not the best way to solve this particular problem. It forces you to keep track of animated properties in two places and always sync the RESET track manually with any changes you make outside of animations. This seems cumbersome and error-prone.
